js/package.json

```diff
@@ -14,7 +14,10 @@
         "rpc"
     ],
     "type": "module",
-    "main": "src/index.js",
+    "exports": {
```

Comment:
Exports is recommended for packages targeting modern NodeJS versions.

The new exported script allows to

```
import { slice2js, slicePath, compile } from "ice/slice2js";
```

`slice2js` is the full path to the slice2js for the given platform.
`slicePath` is the full path to the directory containing the Slice files, which are also included in the package.
`compile` allows to run the compiler using `child_process.spawn` we previously provided this function in the npm slice2js package.